pre-commit/action.yaml

```
@@ -18,8 +18,23 @@ runs:
       with:
         go-version: ^1.16
 
+    - name: Install pre-commit
+      run: |
+        python -m pip install pre-commit
+      shell: bash
+
     - name: Run pre-commit
-      uses: pre-commit/action@v2.0.3
-      with:
-        extra_args: --all-files --config ${{ inputs.pre-commit-config }}
-        token: ${{ inputs.token }}
+      run: |
+        pre-commit run -a
+      shell: bash
+
+    - name: Push pre-commit fixes
+      # Prevent a workflow from stopping when pre-commit fails
```

[IMO] I prefer this comment.

```suggestion
      # Push changes even when pre-commit fails
```